src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Remote/IRemoteSpellCheckService.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.ExternalAccess.Razor;
+
+namespace Microsoft.CodeAnalysis.Razor.Remote;
+
+internal interface IRemoteSpellCheckService
+{
+    ValueTask<int[]> GetSpellCheckRangeTriplesAsync(
```

Comment:
why `Triples` in name `GetSpellCheckRangeTriplesAsync`

Reply:
The result for spell checking is an int array, but every 3 elements in the int array is 3 different bits of data for a single range. Semantic tokens is similar, but has 5 element groups.

Reply:
I see. thanks